applications/GeoMechanicsApplication/tests/cpp_tests/custom_utilities/test_process_utilities.cpp

```diff
@@ -141,4 +150,95 @@ static const std::vector<NamedFactory> ProcessesAndOperations = {
 
 INSTANTIATE_TEST_SUITE_P(ProcessAndOperationUtilitiesTests, ModelPartsTest, ::testing::ValuesIn(ProcessesAndOperations));
 
+KRATOS_TEST_CASE_IN_SUITE(AddProcessesSubModelPartList_AddsEmptyList, KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    // Set
+    const auto project_parameters = Parameters{};
+    Parameters solver_settings(R"(
+        {
+            "model_part_name": "Main"
+        })");
+
+    // Act
+    ProcessUtilities::AddProcessesSubModelPartList(project_parameters, solver_settings);
+
+    // Assert
+    KRATOS_EXPECT_TRUE(solver_settings.Has("processes_sub_model_part_list"))
+    KRATOS_EXPECT_TRUE(solver_settings["processes_sub_model_part_list"].IsArray())
+    KRATOS_EXPECT_TRUE(solver_settings["processes_sub_model_part_list"].size() == 0)
+}
+
+KRATOS_TEST_CASE_IN_SUITE(AddProcessesSubModelPartList_AddsFilledList, KratosGeoMechanicsFastSuiteWithoutKernel)
```

Comment:
Also this test case name we can make slightly more accurate, e.g.:
```suggestion
KRATOS_TEST_CASE_IN_SUITE(AddProcessesSubModelPartList_AddsFilledListBasedOnConstraintsLoadsAndAuxiliaryProcessesOnly, KratosGeoMechanicsFastSuiteWithoutKernel)
```

Reply:
changed